packages/embark-ui/src/components/Transactions.js

```diff
@@ -15,38 +15,51 @@ const Transactions = ({transactions, contracts, changePage, currentPage, numberO
           <h2>Transactions</h2>
         </CardHeader>
         <CardBody>
-          {transactions.map(transaction => (
-            <div className="explorer-row border-top" key={transaction.hash}>
-              <CardTitleIdenticon id={transaction.hash}>Transaction&nbsp;
-                <Link to={`/explorer/transactions/${transaction.hash}`}>
-                  {transaction.hash}
-                </Link>
-              </CardTitleIdenticon>
-              <Row>
-                <Col>
-                  <DebugButton transaction={transaction} contracts={contracts} />
-                </Col>
-              </Row>
-              <Row>
-                <Col md={6}>
-                  <strong>Block number</strong>
-                  <div>{transaction.blockNumber}</div>
-                </Col>
-                <Col md={6}>
-                  <strong>From</strong>
-                  <div>{transaction.from}</div>
-                </Col>
-                <Col md={6}>
-                  <strong>To</strong>
-                  <div>{transaction.to}</div>
-                </Col>
-                <Col md={6}>
-                  <strong>Type</strong>
-                  <div>{transaction.to ? "Contract Call" : "Contract Creation"}</div>
-                </Col>
-              </Row>
-            </div>
-          ))}
+          {transactions
+           .filter(transaction => {
+             const to = transaction.to;
+             if (!to) return true;
```

Comment:
Will this still show contract creation transactions for silent contracts?

Reply:
Good question. I haven't seen such tx in the output previously, and this filter adds a restriction to what's displayed, so I don't think it will show anything extra. But that makes me wonder why those transactions weren't seen previously.

Reply:
I do remember seeing the contract creation txs in the past.

Reply:
No, it won't hide contract creations, because creations have no `to`

Reply:
Is it possible to derive somehow the address of the created contract from the transaction that created it?

Reply:
I know the transaction receipt is supposed to have `contractAddress`, but in our case, I think we just have a simplified receipt. See here: https://web3js.readthedocs.io/en/1.0/web3-eth.html#gettransactionreceipt

Reply:
Calculations for pagination/fetching of transactions is presently in need of revision, similar to how the calculations for blocks were revised in #1427.  Once they're made, we'll see contract creation transactions for the silent contracts.

To filter them out, we can:

(1) get server-side the transaction receipt for each transaction and store it in a `receipt:` key on the transaction object; will need some revisions in an endpoint of blockchain_connector.

(2) use `transaction.receipt.contractAddress` to match against `contract.address` in the case that `transaction.to` is falsy.